cli_tools/import_precheck/precheck/check.go

```diff
@@ -0,0 +1,24 @@
+//  Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
minor: 2021. For other new files as well, unless they were just moved.

Reply:
No new files in this PR -- this was moved from `cli_tools/import_precheck/check.go`, for example.